src/Commands/stubs/event.stub

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace $NAMESPACE$;
+
+use $DUMMYNAMESPACE$\Event;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
```

Comment:
Hm I think so yea, and people can implement it as needed. Otherwise the IDE like phpstorm will say it's not used and hint to remove it. :)